### DISCUSSION TITLE
Possibility to add new seed data from cli or UI

### DISCUSSION THREAD

**Discussion by Bananenbrot1** [supabase/supabase#15945] [Feature Requests]
Hey,

It would be cool if there is a small tool, that takes content from one or multiple tables and dumps it into the seed file. This way, devs could easily create seed data, while testing the system and then release it for other devs to reuse.

**Comment by silentworks** [comment]
Isn't this just pg_dump and then pipe that data into a seed.sql file?

```sql
pg_dump --column-inserts --data-only --table=<table> <database>
```
